src/Middleware/ErrorResponseGenerator.php

```diff
@@ -62,14 +62,15 @@ public function __invoke(
     ) : ResponseInterface {
         $response = $response->withStatus(Utils::getStatusCode($e, $response));
 
-        if ($this->renderer) {
-            return $this->prepareTemplatedResponse($e, $request, $response);
+        if ($this->renderer !== null) {
+            return $this->prepareTemplatedResponse($this->renderer, $e, $request, $response);
```

Comment:
As above, it looks quite weird to pass class property to the private class method

Reply:
Same thing: can't rely on the property due to it being documented as nullable, so it is added as a call-time parameter once it is checked